cfgov/legacy/templates/complaint/complaint-landing.html

```diff
@@ -35,13 +35,147 @@
+                  <span class="a-link_text">See publication criteria</span>
+                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 651.7 1200" class="cf-icon-svg"><path d="M507.1 692.8c-15.6-15.6-40.9-15.6-56.6 0l-85.1 85.1V466.6c0-22.1-17.9-40-40-40s-40 17.9-40 40V778l-85.1-85.1c-15.6-15.6-40.9-15.6-56.6 0-15.6 15.6-15.6 40.9 0 56.6L297 902.9c7.5 7.5 17.7 11.7 28.3 11.7s20.8-4.2 28.3-11.7l153.3-153.4c15.8-15.7 15.8-41 .2-56.7z"/><path d="M30 161c-16.5 0-30 13.5-30 30v827.8c0 16.5 13.5 30 30 30h591.7c16.5 0 30-13.5 30-30V343.7L469 161H30zm389.6 60v134.8c0 19.9 16.3 36.2 36.2 36.2h135.9V988.8H60V221h359.6z"/></svg>
+                </a>
+              </span>
+            </p>
+            <p>Have an issue with a financial product or service? We have
+              <a href="/ask-cfpb/">
+                answers to frequently asked questions
+              </a>
+              and can help you connect with companies if you have a complaint.
+              <a href="/complaint/">
+                Learn more
+              </a>
+            </p>
+            <h2 id="what-you-should-consider">What you should consider when using the data</h2>
+            <p>This database is not a statistical sample of consumers’ experiences in the marketplace. Complaints are not necessarily representative of all consumers’ experiences and complaints do not constitute “information” for purposes of the
+              <a href="/open-government/information-quality-guidelines/">
+                Information Quality Act
+              </a>.
+            </p>
+            <p>Complaint volume should be considered in the context of company size and/or market share. For example, companies with more customers may have more complaints than companies with fewer customers. We encourage you to pair complaint data with public and private data sets for additional context.</p>
+            <p>The Bureau publishes the consumer’s narrative description of his or her experience if the consumer opts to share it publicly and after the Bureau takes steps to remove personal information. We don’t verify all the allegations in complaint narratives. Unproven allegations in consumer narratives should be regarded as opinion, not fact. We do not adopt the views expressed and make no representation that consumers’ allegations are accurate, clear, complete, or unbiased in substance or presentation. Users should consider what conclusions may be fairly drawn from complaints alone.</p>
+            <h2>How we use complaint data</h2>
+            <p>Complaints can give us insights into problems people are experiencing in the marketplace and help us regulate consumer financial products and services under existing federal consumer financial laws, enforce those laws judiciously, and educate and empower consumers to make informed financial decisions. We also report on complaint trends annually in
```

Comment:
Move the text to the next line so it aligns with the link indent.

Reply:
Fixed